kura/org.eclipse.kura.linux.watchdog/src/main/java/org/eclipse/kura/linux/watchdog/WatchdogServiceImpl.java

```diff
@@ -72,7 +74,7 @@ public void updated(Map<String, Object> properties) {
         if (this.options.isEnabled()) {
             this.rebootCauseWriter = new RebootCauseFileWriter(this.options.getRebootCauseFilePath());
 
-            try (PrintWriter wdWriter = new PrintWriter(this.options.getWatchdogDevice());) {
+            try (PrintWriter wdWriter = new PrintWriter(WD_FILE);) {
```

Comment:
Great. But at this point, since we have the properties class, I would move the WD_FILE there. Furthermore, in `WatchdogServiceOptions` the default for `PROPERTY_WD_DEVICE` is `"/tmp/watchdog"` that does not match the default value in the metatype.